creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -243,6 +247,40 @@ async function _handleIssueSyncRequest({
       syncMode === SYNC_MODES.MergePrimaryAndSecondary ||
       syncMode === SYNC_MODES.MergePrimaryThenWipeSecondary
     ) {
+      // Ensure this node is the user's primary making a request to one of the user's secondaries
```

Comment:
question - did you intentionally not use `ensurePrimaryMiddleware` on the route controller?

also you enforce that the request is being made to a user's secondary but the orphaned node is explicitly not a secondary? how does that work

Reply:
1. makes sense for /merge_primary_and_secondary route - added now

2. lol wow my test was only for when the node wasn't the primary but missed a test for issuing to non-secondary. great catch - fixed!

Reply:
sick